sdks/python/apache_beam/io/filesystem.py

```diff
@@ -269,8 +269,9 @@ class FileMetadata(object):
   """
   def __init__(self, path, size_in_bytes):
     assert isinstance(path, basestring) and path, "Path should be a string"
-    assert isinstance(size_in_bytes, int) and size_in_bytes >= 0, \
-        "Size of bytes should be greater than equal to zero"
+    assert isinstance(size_in_bytes, (int, long)) and size_in_bytes >= 0, \
+        "Size of bytes should be greater than zero %s with type %s"  % (
```

Comment:
Was not addressed: 	
This comment right now incorrectly suggests that the printed type(size_in_bytes) is the desired type. How about the following: `"Invalid value for size_in_bytes: %s (of type %s)"`